recipes/carbonplan/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "carbonplan" %}
-{% set version = "0.5.0" %}
+{% set version = "0.6.0" %}
```

Comment:
You need to update the checksum for this version. You can get the sha256 from PyPI, or you can use grayskull to regenerate the recipe.

Reply:
hmm I just checked PyPI and the checksum looks to be accurate for 0.6.0 below

Reply:
That comment was addressed. The new failure is in the logs: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=939714&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

Reply:
Ah, right. That should be fixed by the code suggestions above, which bump the version to 0.4.4